logprep/connector/http/input.py

```diff
@@ -432,8 +461,32 @@ class Config(Input.Config):
         be smaller than default value of 15.000 messages.
         """
 
-        collect_meta: str = field(validator=validators.instance_of(bool), default=True)
-        """Defines if metadata should be collected
+        copy_headers_to_logs: Set[str] = field(
+            validator=validators.deep_iterable(
+                member_validator=validators.instance_of(str),
+                iterable_validator=validators.or_(
+                    validators.instance_of(set), validators.instance_of(list)
+                ),
+            ),
+            converter=set,
+            default=set(DEFAULT_META_HEADERS),
```

Comment:
```suggestion
            default=DEFAULT_META_HEADERS,
```

Should be fine as this is a frozenset which cannot be modified